Source/Core/Component/Component.swift

```diff
@@ -27,15 +27,47 @@ public protocol Component: Invalidable {
     associatedtype StateType
     associatedtype ActionType
 
+    /**
+     * This `DisposeBag` gets disposed at `Component`'s **deinit** and should be used
+     * to dispose subscriptions outside of `update()`.
+     *
+     * **Usage**:
+     * ```
+     * // inside afterInit()
+     *
+     * friendService.defaultFriends
+     *   .subscribe(onNext: { friends in
+     *     componentState = friends
+     *   }
+     *   .disposed(by: lifetimeDisposeBag)
+     * ```
+     *
+     * - WARNING: It's strongly discouraged to use this `DisposeBag` in the `update()` method.
+     * Use the `stateDisposeBag` for that.
+     */
     /**
```

Comment:
Is leaving the old reference comment here intentional?

Reply:
Nope, I was comparing these two to see what should be added to the new one and forgot to remove it.